css/build.css

```
@@ -144,4 +144,8 @@
 /* For native devices */
 .native .fa-times-thin {
   padding-top: 4px;
-} 
\ No newline at end of file
+}
+
+.dynamic-list-layout {
```

You can just use

```suggestion
[data-dynamic-lists-id] {
```

I don't think a new class is necessary.